routes/main.js

```diff
@@ -1,12 +1,12 @@
-const express = require('express')
-const { getAllUrls, getUrl, editUrl, deleteUrl } = require('../controllers/main')
+const express = require('express');
+const { getAllUrls, getUrl, editUrl, deleteUrl } = require('../controllers/main');
+const { validateUrl, validateUrlMiddleware } = require('../middleware/validation');
 
-router = express.Router()
+router = express.Router();
 
+router.route('/').get(getAllUrls);
+router.route('/:id').get(getUrl);
+router.route('/:id').put(validateUrl, validateUrlMiddleware, editUrl);
+router.route('/:id').delete(deleteUrl);
```

Comment:
Can you use the same route format as [urls](https://github.com/PraiseTy/Short-It/blob/67506063906b8926bdb6519a96de4bec6d0eec3b/routes/urls.js#L7-L8) file?

Reply:
I'm not sure I understand what you mean as the same format

Reply:
Here, you use `router.route` to define the url and separate the handlers like this `router.route('/:id').delete(deleteUrl)`.

In the order file, you define the URL with the handler together without `.route` like this `router.post('/', customShortUrl)`. 

You have to be consistent with how you write code. There is no need to use two separate approaches in the same codebase. I prefer the URL with the handlers together like the one [here](https://github.com/PraiseTy/Short-It/blob/67506063906b8926bdb6519a96de4bec6d0eec3b/routes/urls.js#L7-L8), I'll suggest you use that.

Reply:
Noted. Done on ff6ae1d